TypeScript/index.js

```diff
@@ -4,61 +4,59 @@ module.exports = {
     'tslint:recommended',
   ],
   rules: {
-    // 들여쓰기는 공백 2칸을 사용합니다.
+    // 들여쓰기는 공백 2칸을 사용한다.
     'indent': {
       options: [
         'spaces',
         2,
       ],
     },
 
-    // 인터페이스에서 I 접두사를 붙이지 않습니다.
+    // 인터페이스에서 I 접두사를 붙이지 않는다.
     'interface-name': {
       options: [
         'never-prefix',
       ],
     },
 
-    // 한 파일에서 여러개의 클래스를 허용합니다.
+    // 한 파일에서 여러개의 클래스를 허용한다.
     'max-classes-per-file': {
       options: [
         10,
       ],
     },
 
-    // 한 줄은 한 화면에 들어오게 하는 것이 권장사항입니다.
-    // 다만 특별한 경우에는 길게 작성해도 됩니다.
+    // 한 줄은 한 화면에 들어오게 하는 것이 권장사항이다.
+    // 다만 특별한 경우에는 길게 작성해도 된다.
```

Comment:
```suggestion
    // 다만 특별한 경우에는 길게 작성하는 것을 허용한다.
```
이렇게 쓰는 것이 다른 부분과 일관성 있어보입니다.